public/app/core/components/search/search.ts

```diff
@@ -0,0 +1,355 @@
+import _, { debounce } from 'lodash';
```

Comment:
Why are there 2 new components?

Reply:
Oops, looks like it got re-added while merging changes - it was already removed. Gonna remove.